kong/plugins/cors/handler.lua

```diff
@@ -181,7 +181,9 @@ end
 
 
 function CorsHandler:access(conf)
-  if kong.request.get_method() ~= "OPTIONS" then
+  if not (kong.request.get_method() == "OPTIONS"
```

Comment:
style: please remove the extra set of parenthesis around this expression. They are not necessary.